source/common/upstream/load_stats_reporter.cc

```diff
@@ -109,15 +118,29 @@ void LoadStatsReporter::onReceiveMessage(
 }
 
 void LoadStatsReporter::startLoadReportPeriod() {
+  // Once a cluster is tracked, we don't want to reset its stats between reports
+  // to avoid racing between request/response.
+  std::unordered_map<std::string, std::chrono::steady_clock::duration> existing_clusters;
```

Comment:
Given this is temporary, is it possible to avoid all the extra string creation, or is it more trouble than it's worth for today?